src/lib/teads-aws/index.ts

```diff
@@ -1,148 +1,109 @@
 import Spline from 'typescript-cubic-spline';
+import {z} from 'zod';
 
-import * as AWS_INSTANCES from './aws-instances.json';
-import * as AWS_EMBODIED from './aws-embodied.json';
+import {ModelPluginInterface} from '../../interfaces';
+import {Interpolation, KeyValuePair, ModelParams} from '../../types/common';
+import {IComputeInstance} from '../../types';
```

Comment:
since we are not using I<interfacename> pattern, I from the beginning can be dropped

Reply:
done